.github/workflows/scripts/changelog-release-helper.test.mjs

```diff
@@ -73,6 +78,25 @@ describe('Changelog release helper', () => {
         expect.stringContaining('## v3.1.0 (Feature release)')
       )
     })
+
```

Comment:
Let's add a test for incrementing a pre-release (like `3.1.0-beta.1` to `3.1.0-beta.2`)